packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -3,6 +3,8 @@ import { UserValidationRule } from "./user-validation-rule.enum.js";
 const UserValidationMessage = {
 	EMAIL_REQUIRED: "Email is required.",
 	EMAIL_WRONG: "Email is wrong.",
+	NAME_REQUIRED: "Name is required.",
+	NAME_TOO_LONG: "Name is too long (> 100 symbols).",
```

Comment:
NAME_TOO_LONG: `Name is too long (> ${UserValidationRule.NAME_MAXIMUM_LENGTH} symbols).`,